packages/forms/src/UIForm/fields/Enumeration/EnumerationWidget.test.js

```diff
@@ -258,7 +257,9 @@ describe('EnumerationWidget', () => {
 
 		// then
 		expect(onTrigger).toBeCalledWith(expect.anything(), {
-			schema: {},
+			schema: {
+				skipCommas: true,
```

Comment:
you need to update the previous test as well https://github.com/Talend/ui/blob/master/packages/forms/src/UIForm/fields/Enumeration/EnumerationWidget.test.js#L250